test/taskrun_test.go

```diff
@@ -126,13 +127,17 @@ func TestTaskRunStatus(t *testing.T) {
 	taskRunName := "status-taskrun"
 
 	fqImageName := "busybox@sha256:895ab622e92e18d6b461d671081757af7dbaa3b00e3e28e12505af7817f73649"
+	if runtime.GOARCH == "s390x" {
+		fqImageName = "busybox@sha256:4f47c01fa91355af2865ac10fef5bf6ec9c7f42ad2321377c21e844427972977"
```

Comment:
We have quite a lot of places that uses the busybox image around, maybe we can have a common function which could be extended easily for other architectures if needed ?

Reply:
Fix is required only for places where busybox sha is specified directly. I identified 1 piece of code for e2e tests(fixed by this PR), and 2 pieces for examples.
Just busybox image is multi-arched, and expectation here, that if you have `image: busybox` statement it should run on all architectures without modifications.
Anyway, it's a good point for other cases (and can be applied even here), where image name for amd64 will be different to s390x or ppx64le names. Thank you for feedback, I will work on this extension.

Reply:
awesome thank you! 🙇